src/libraries/Common/src/System/Security/Cryptography/Asn1/AsnXml.targets

```diff
@@ -1,16 +1,59 @@
 <Project>
   <PropertyGroup>
     <AsnXml />
-    <_AsnXmlDiffCmd>%24%28command -v diff%29 -q -a</_AsnXmlDiffCmd>
-    <_AsnXmlDiffCmd Condition="$([MSBuild]::IsOsPlatform('windows')) == 'true'">$(SystemRoot)\System32\fc.exe /a</_AsnXmlDiffCmd>
-    <_AsnXmlDiffCmd Condition="$([MSBuild]::IsOsPlatform('osx')) == 'true'">diff</_AsnXmlDiffCmd>
   </PropertyGroup>
   <ItemGroup>
     <None Include="$(MSBuildThisFileDirectory)asn.xsd">
       <Link>Common\System\Security\Cryptography\Asn1\asn.xsd</Link>
     </None>
   </ItemGroup>
 
+  <UsingTask
+    TaskName="CompareFilesIgnoreLineEndings"
+    TaskFactory="RoslynCodeTaskFactory"
+    AssemblyFile="$(MSBuildToolsPath)\Microsoft.Build.Tasks.Core.dll">
+    <ParameterGroup>
+      <GeneratedFile ParameterType="Microsoft.Build.Framework.ITaskItem" Required="true" />
+      <ExistingFile ParameterType="Microsoft.Build.Framework.ITaskItem" Required="true" />
+      <ContentsDiffer ParameterType="System.Boolean" Output="true" />
+    </ParameterGroup>
+    <Task>
+      <Using Namespace="System" />
+      <Using Namespace="System.Collections.Generic" />
```

Comment:
By replacing these two with `<Using Namespace="System.Linq" />`, fragment code can be simplified without compromising memory optimization:
```c#
ContentsDiffer = !File.Exists(ExistingFile.ItemSpec) || File.ReadLines(ExistingFile.ItemSpec)
    .SequenceEqual(File.ReadLines(GeneratedFile.ItemSpec));
return true;
```

Reply:
Thanks. I don't know why my brain reverted to 2004 when writing an MSBuild task definition in C#.